install.js

```diff
@@ -27,10 +27,11 @@ function onerror(err) {
 function install(runtime, abi, platform, arch, cb) {
   const essential = runtime + '-v' + abi + '-' + platform + '-' + arch;
   const pkgVersion = pkg.version;
-  const currentPlatform = pkg.name + '-v' + pkgVersion + '-' + essential;
+  const currentPlatform = 'iohook-v' + pkgVersion + '-' + essential;
 
   console.log('Downloading prebuild for platform:', currentPlatform);
-  let downloadUrl = 'https://github.com/wilix-team/iohook/releases/download/v' + pkgVersion + '/' + currentPlatform + '.tar.gz';
+  const url = github(pkg)
```

Comment:
```suggestion
  const url = pkg.homepage;
```
This won't work with your base but once rebased will work.